pallets/subtensor/src/macros/hooks.rs

```diff
@@ -41,6 +41,7 @@ mod hooks {
         // 		- The number of the block we are finalizing.
         fn on_finalize(block_number: BlockNumberFor<T>) {
             Self::do_on_finalize(block_number);
+            Self::clean_up_hotkey_swap_records(block_number);
```

Comment:
Probably better to do this in `on_finalize` except when absolutely needed, wdyt?